Chaos Dashboard panics on invalid memory address or nil pointer dereference

## Bug Report

```
2020-06-22T16:21:05.439Z        INFO    database ttl controller deleting expired data from the database
E0622 16:21:05.762886       1 runtime.go:78] Observed a panic: "invalid memory address or nil pointer dereference" (runtime error: invalid memory address or nil pointer derefe
rence)
goroutine 156 [running]:
k8s.io/apimachinery/pkg/util/runtime.logPanic(0x187e6a0, 0x2af1530)
        /go/pkg/mod/k8s.io/apimachinery@v0.17.1-beta.0/pkg/util/runtime/runtime.go:74 +0xa3
k8s.io/apimachinery/pkg/util/runtime.HandleCrash(0x0, 0x0, 0x0)
        /go/pkg/mod/k8s.io/apimachinery@v0.17.1-beta.0/pkg/util/runtime/runtime.go:48 +0x82
panic(0x187e6a0, 0x2af1530)
        /usr/local/go/src/runtime/panic.go:969 +0x166
github.com/pingcap/chaos-mesh/pkg/store/event.(*eventStore).DeleteByFinishTime(0xc0002085d8, 0x1e687c0, 0xc000046028, 0x2260ff9290000, 0x0, 0x0)
        /src/pkg/store/event/event.go:293 +0xf7
github.com/pingcap/chaos-mesh/pkg/ttlcontroller.(*Controller).runWorker(0xc0013117c0)
        /src/pkg/ttlcontroller/ttlcontroller.go:72 +0x97
k8s.io/apimachinery/pkg/util/wait.JitterUntil.func1(0xc00044b320)
        /go/pkg/mod/k8s.io/apimachinery@v0.17.1-beta.0/pkg/util/wait/wait.go:152 +0x5f
k8s.io/apimachinery/pkg/util/wait.JitterUntil(0xc00044b320, 0x274a48a78000, 0x0, 0x1, 0xc00047e0c0)
        /go/pkg/mod/k8s.io/apimachinery@v0.17.1-beta.0/pkg/util/wait/wait.go:153 +0xf8
k8s.io/apimachinery/pkg/util/wait.Until(0xc00044b320, 0x274a48a78000, 0xc00047e0c0)
        /go/pkg/mod/k8s.io/apimachinery@v0.17.1-beta.0/pkg/util/wait/wait.go:88 +0x4d
created by github.com/pingcap/chaos-mesh/pkg/ttlcontroller.Register
        /src/pkg/ttlcontroller/ttlcontroller.go:65 +0xf1
panic: runtime error: invalid memory address or nil pointer dereference [recovered]
        panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x10 pc=0x15fd3d7]
```

--- a/pkg/store/event/event.go
+++ b/pkg/store/event/event.go
@@ -320,6 +320,9 @@ func (e *eventStore) DeleteByFinishTime(_ context.Context, ttl time.Duration) er
 	}
 	nowTime := time.Now()
 	for _, et := range eventList {
+		if et.FinishTime == nil {
+			continue
+		}
 		if et.FinishTime.Add(ttl).Before(nowTime) {
 			if err := e.db.Model(core.Event{}).Unscoped().Delete(*et).Error; err != nil {
 				return err

